tests/integration_tests.rs

```diff
@@ -1,17 +1,16 @@
 extern crate bitcask;
-
-use bitcask::tests::common;
-
 use bytesize::ByteSize;
 
+mod common;
+
 #[test]
 fn writing_a_key_should_return_same_value() {
     let mut db = common::DatabaseTesting::new("db1".to_owned(), ByteSize::b(1).as_u64());
 
-    db.write("name".as_bytes(), "Peter".as_bytes()).unwrap();
+    db.write(b"name", b"Peter").unwrap();
```

Comment:
`&'static str::as_bytes()` can be a bytestring literal (`b"foobar"`) instead.

up until very recently, `str::as_bytes()` wasn't const so there could be some overhead introduced here.